bond_app/fence_api.py

```diff
@@ -37,7 +37,9 @@ def delete_credentials_google(self, access_token, key_id):
     def revoke_refresh_token(self, refresh_token):
```

Comment:
Looking at the code, I'm not sure we ever call this method.  I think we only ever use the implementation in `OauthAdapter`

Reply:
Yeah, I changed it here for consistency but couldn't actually figure out what `fence_api.py` was for or whether I needed to change this function at all